marathon-gradle-plugin/build.gradle.kts

```diff
@@ -36,17 +36,14 @@ dependencies {
 }
 
 tasks.processResources.configure {
-    from(rootProject.project("cli").layout.buildDirectory.dir("distributions").get().asFile) {
+    val zipTask: Task = rootProject.project("cli").tasks.getByName("distZip")
```

Comment:
Explicitly specify marathon cli dist, otherwise updating version doesn't pick up the new binary on local builds